js/entryView.js

```diff
@@ -17,7 +17,6 @@
       description: $('#entry-description').val()
     });
     $('#entries').append(entry.toHtml());
```

Comment:
entryView.create function isnt being called because handlebars are populating articles to the page. 
It can be deleted.